From: Fabian Vogt <fabian@ritter-vogt.de>
Subject: Skip building the NFS KIO slave

It fails to build with glibc 2.27:

[  164s] -- Looking for xdr_u_int64_t
[  164s] -- Looking for xdr_u_int64_t - not found
[  164s] -- Looking for xdr_uint64_t
[  165s] -- Looking for xdr_uint64_t - not found
[  165s] -- Looking for xdr_u_hyper
[  165s] -- Looking for xdr_u_hyper - not found
[  165s] -- Looking for xdr_u_longlong_t
[  166s] -- Looking for xdr_u_longlong_t - not found
[  166s] CMake Error at kioslave/nfs/CMakeLists.txt:19 (message):
[  166s]   Could not find 64-bit XDR datatype functions!

Index: kde-runtime-17.08.3/kioslave/CMakeLists.txt
===================================================================
--- kde-runtime-17.08.3.orig/kioslave/CMakeLists.txt
+++ kde-runtime-17.08.3/kioslave/CMakeLists.txt
@@ -55,7 +55,7 @@ if(NOT WIN32)
    check_include_files(rpc/rpc.h HAVE_RPC_RPC_H)
    add_feature_info("NFS kioslave" HAVE_RPC_RPC_H "The RPC library is needed to build the NFS kioslave")
    if(HAVE_RPC_RPC_H)
-     add_subdirectory( nfs )
+#    add_subdirectory( nfs )
    endif(HAVE_RPC_RPC_H)
 endif(NOT WIN32)
 
